specification/synapse/data-plane/Microsoft.Synapse/preview/2022-03-01-preview/runNotebook.json

```diff
@@ -0,0 +1,562 @@
+      "type": "object",
+      "properties": {
+        "exitValue": {
+          "description": "Output of exit command.",
+          "type": "string"
+        },
+        "id": {
+          "description": "Run notebook runId.",
+          "type": "string"
+        },
+        "notebook": {
+          "description": "Notebook name.",
+          "type": "string"
+        },
+        "sessionOptions": {
+          "description": "Session properties.",
+          "$ref": "#/definitions/RunNotebookSparkSessionOptions"
+        },
+        "honorSessionTimeToLive": {
+          "description": "Whether session should run till time to live after run completes.",
+          "type": "boolean"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        },
+        "parameters": {
+          "type": "object",
+          "description": "Run notebook parameters",
+          "additionalProperties": {
+            "$ref": "#/definitions/RunNotebookParameter"
+          }
+        },
+        "notebookContent": {
+          "$ref": "../2021-06-01-preview/notebooks.json#/definitions/NotebookResource"
+        }
+      },
+      "required": [
+        "id",
+        "notebook"
+      ]
+    },
+    "RunNotebookSnapshotResult": {
+      "description": "Run notebook snapshot result.",
+      "type": "object",
+      "properties": {
+        "snapshot": {
+          "$ref": "#/definitions/RunNotebookSnapshot"
+        },
+        "error": {
+          "$ref": "#/definitions/RunNotebookError"
+        },
+        "runId": {
+          "type": "string",
+          "description": "Run id."
+        },
+        "runStatus": {
+          "description": "Status of the run notebook.",
+          "type": "string"
+        },
+        "lastCheckedOn": {
+          "description": "Timestamp of last update.",
+          "type": "string"
+        },
+        "sessionId": {
+          "description": "Livy session id.",
+          "type": "integer",
+          "format": "int64"
+        },
+        "sparkPool": {
+          "description": "SparkPool name.",
+          "type": "string"
+        }
+      },
+      "required": [
+        "runId",
+        "runStatus",
+        "snapshot"
+      ]
+    },
+    "RunNotebookSnapshotResponse": {
+      "description": "Run notebook snapshot response.",
+      "type": "object",
+      "properties": {
+        "message": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Response message."
+        },
+        "result": {
+          "$ref": "#/definitions/RunNotebookSnapshotResult"
+        }
+      }
+    },
+    "RunNotebookSparkSessionOptions": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "x-nullable": true,
```

Comment:
You should avoid the use of `x-nullable: true`. Properties with no value should simply be omitted from the payload.

Reply:
`x-nullable: true` has been removed now